db/schema.rb

```diff
@@ -10,27 +10,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190905130441) do
+ActiveRecord::Schema.define(version: 20190921120240) do
 
-  create_table "airplanes", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
+  create_table "airplanes", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci" do |t|
```

Comment:
This `id: :integer` was added by Rails because Rails now defaults to BigInt for `id` columns. The integer is still plenty large enough for this application, and is a legacy. The BigInt might be worthwhile on some of the tables, but not for a long time.